api_app/analyzers_manager/file_analyzers/pe_info.py

```diff
@@ -12,10 +12,79 @@
 import pefile
 
 from api_app.analyzers_manager.classes import FileAnalyzer
+from api_app.exceptions import AnalyzerRunException
 
 logger = logging.getLogger(__name__)
 
 
+class WinPEhasher:
+    file_path: str
+
+    def __init__(self, file_path: str) -> None:
+        """
+        Init a WinPE instance and load the exe file
+        """
+        self.file_path = file_path
+
+    def dhashicon(self) -> str:
+        """
+        Return Dhashicon of the exe
+        """
+        try:
+            import os
+
+            import lief
+            from PIL import Image
+
+            # config
+            hash_size = 8
```

Comment:
yeah well the code is copy-pasted so we should write down reference to where we got the original code here.

Reply:
Ok Sir. I'll add it.